### PR TITLE
Support leaving ClosedCaption styles up to the app

### PR DESCRIPTION
**Pull request by SKumarMetro**
## What
Support leaving ClosedCaption styles up to the app

## Why
R2 sync

## How
closed caption is in sync with ripple 1.0

## Test
Using python script making ws request to ripple and verified all the modified APs.

## Checklist

- [ x] I have self-reviewed this PR
- [ x] I have added tests that prove the feature works or the fix is effective

### CODE REVIEW

**Comment by satlead** [comment]
@SKumarMetro  can we remove the RPPL references from title. Going forward lets not use the branch names with RPPL

File: device/thunder_ripple_sdk/src/processors/thunder_persistent_store.rs [hunk 1]
```diff
@@ -78,30 +82,36 @@ impl ThunderStorageRequestProcessor {
...
             })
             .await;
-        if response.message.get("success").is_none()
-            || !response.message["success"].as_bool().unwrap_or_default()
-        {
-            error!("{}", response.message);
-            return false;
-        }
-        true
+        info!("{}", response.message);
```
Comment (satlead) [inline device/thunder_ripple_sdk/src/processors/thunder_persistent_store.rs:105]: Do we need these logs in info level?

**Review by satlead — COMMENTED** [review]
This change is `ready-for-qa` needs minor logging comments and PR title to be fixed

File: device/thunder_ripple_sdk/src/processors/thunder_persistent_store.rs (hunk 1 above)
Comment (SKumarMetro) [reply device/thunder_ripple_sdk/src/processors/thunder_persistent_store.rs]: Nope, removed it.

**Comment by satlead** [comment]
These changes are related to Firebolt 16 version and needs to be verified by QA before proceeding. cc @dhillomk

**Comment by satlead** [comment]
> These changes are related to Firebolt 16 version and needs to be verified by QA before proceeding. cc @dhillomk

Not applicable anymore

**Review by satlead — APPROVED** [review]

**Review by Nv-Delle1416 — APPROVED** [review]

**Pull request merged**